Documentation/Storage-Configuration/requirements.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
We should add a link to the pre-requisites page in the Getting Started section.

Reply:
Changes updated.

Reply:
It should be both ways, as both the pre-requisites and these storage requirements are a must, it might even make sense to combine both pages to some degree or moving this one to the Getting Started section and renaming it to "Storage Requirements" or something else fitting.

Reply:
@galexrt ah yeah i think moving this to Getting started section makes sense because it seems like a must to have.

Reply:
will make some quick changes and update here.